docs/Workflow.md

```diff
@@ -1,39 +1,83 @@
 <!--- SPDX-License-Identifier: Apache-2.0 -->
 # Contribution Guide
 
-## Prerequisite: Install LLVM
+## Step 1: Fork ONNX-MLIR on the GitHub web interface
 
-Follow directions in this section of the top level [README](../README.md#mlir) 
-for installing the currently supported LLVM version. We assume here a
-LINUX installation. 
+We strongly encourage contributors to work in their own forks of the ONNX-MLIR project.
 
-### Step 1: Fork in the cloud
+Creating a new fork is easy:
 
 1. Visit https://github.com/onnx/onnx-mlir
 2. Click `Fork` button (top right) to establish a fork.
+3. Navigate to your newly created fork, click on the the green `Code` button to get the link to *your* newly-created ONNX-MLIR fork:
+```sh
+git@github.com:<user>/onnx-mlir.git
+```
+or
+```sh
+https://github.com/<user>/onnx-mlir.git
+```
 
-### Step 2: Clone fork to local storage
+where `<user>` is your GitHub username.
 
-Define a local working directory:
+## Step 2: Setup MLIR
+
+### Setup using Docker
 
+Use the template provided in [here](Docker.md#building-onnx-mlir-in-a-docker-environment) to establish a docker image that uses your ONNX-MLIR fork by modifying it as follows:
```

Comment:
Some contributors may not want (or can't) use docker.  Can we generalize the instruction to also ppl that do not use docker pls.

Reply:
same comment as above.